__tests__/app.test.js

```diff
@@ -63,13 +63,22 @@ describe("GET /api/articles/:article_id", () => {
             body: "Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.",
             created_at: "2020-10-16T05:03:00.000Z",
             votes: 0,
+            comment_count: 0,
           })
```

Comment:
Hello! Please don't change you previous tests to include the comment count. This does not follow good TDD because you are amending your tests to match new behaviour, rather than adding a new test to handle the new behaviour whilst also passing all previous tests. Please revert your original test and add an extra one to handle the comment count